haskell/test/Lorentz/Contracts/Test/Management.hs

```diff
@@ -1023,14 +1020,14 @@ managementSpec originate =
             testMasterMinter <- newAddress "testMasterMinter"
             wallet1 <- newAddress "wallet1"
             stablecoinContract <-
-              originate $
+              originateSc $
                 ( defaultOriginationParams
                     (#owner :! testOwner)
                     (#pauser :! testPauser)
                     (#masterMinter :! testMasterMinter)
                 )
             do
-              mgmNotContractOwner $ withSender testMasterMinter $ call stablecoinContract (Call @"Change_master_minter") wallet1
+              mgmNotContractOwner $ withSender testMasterMinter $ transfer stablecoinContract $ calling (ep @"Change_master_minter") $ toAddress wallet1
```

Comment:
Could you wrap some of these overly long lines?

Reply:
Done. Also removed some redundant `do` blocks, and fused usages of `withSender`.